cycle/docker-compose.yaml

```diff
@@ -480,7 +480,7 @@ services:
       args:
         hub: ts-dockerhub.lsst.org
         base_image_tag: ${CYCLE}
-        ts_dds: ${ts_dds_private}
+        ts_dds: ${ts_dds_private}=${ts_dds_private_conda_build}
```

Comment:
It that really an = sign or should it be a -?

Reply:
yeap.. really is "=".. :)